executables/windows/.gitignore

```diff
@@ -0,0 +1,5 @@
+templates/grr-client*
```

Comment:
templates/unzipsfx isn't getting ignored.  Is there another templates/grr-client*/unzipsfx involved?
I'm not even certain that grr-client* is necessary any more since they all seem to be /grr_* or GRR_*